package.json

```diff
@@ -85,14 +84,13 @@
     "d3-transition": "^1.0.0",
     "d3-zoom": "^1.8.3",
     "dateformat": "^3.0.3",
-    "domhandler": "^2.4.2",
```

Comment:
I think we shouldn't import "domhandler" without having a direct dependency on "domhandler".